pubspec.yaml

```diff
@@ -1,14 +1,15 @@
 name: flutter_dotenv
-version: 3.1.0
+version: 3.2.0-nullsafety.0
 description: Easily configure any flutter application with global variables using a `.env` file.
 author: java-james <james-collins@hotmail.co.nz>
 homepage: https://github.com/java-james/flutter_dotenv
 environment:
-  sdk: '>=2.0.0 <3.0.0'
+  sdk: '>=2.12.0-0 <3.0.0'
```

Comment:
Stable is 2.10?
Consumers running stable will be unable to consume this library if we update to 2.12?
What version was null safety introduced? (Not as experimental)

Reply:
`sdk: '>=2.12.0-0 <3.0.0'` is a requirement for null safety to work. This will break backwards compatibility so it'll have to be a major version bump, with `-nullsafety.0` at the end of the version name. You can have both the stable, non-nullsafe version and the preview, nullsafe version of your package on pub.

Reply:
Reading https://dart.dev/null-safety/migration-guide we need to release this as prerelease if published

Reply:
yes

Reply:
Ah, got it - Reading https://dart.dev/null-safety/migration-guide we need to release this as prerelease

Reply:
Prerelease is not needed anymore actually.
pub.dev now recognizes if a package uses a not yet release dart version. It is then displayed as preview.

![image](https://user-images.githubusercontent.com/1096841/109790484-c1a47180-7c11-11eb-9e32-54158c8f8b07.png)

Reply:
@nstrelow Cool, thanks for the info